test/ci/buildkite/setup.yml

```diff
@@ -1,3 +1,6 @@
+env:
```

Comment:
May be put a comment like: Per BuildKite doc, defining an env var at the top of the yaml file will set that var on each of the command steps in the pipeline, and is equivalent to setting the env attribute on every step.

Reply:
I figured that it was obvious from the context since this is in `setup.yml`. Will document the pipeline scripts some more in follow-up PRs.